tests/pipelines/test_pipelines.py

```diff
@@ -1402,15 +1407,13 @@ def test_output_format(self):
         assert isinstance(images, list)
         assert isinstance(images[0], PIL.Image.Image)
 
+    @require_flax
     def test_from_flax_from_pt(self):
         pipe_pt = StableDiffusionPipeline.from_pretrained(
             "hf-internal-testing/tiny-stable-diffusion-torch", safety_checker=None
         )
         pipe_pt.to(torch_device)
 
-        if not is_flax_available():
-            raise ImportError("Make sure flax is installed.")
-
```

Comment:
Isn't `require_flax` better than the import error? 

For running the Slow PyTorch test suite, we don't expect Flax to be installed. Do we?